Core/include/Acts/Seeding/BinnedSPGroup.hpp

```diff
@@ -266,18 +266,14 @@ class BinnedSPGroup {
  public:
   BinnedSPGroup() = delete;
 
-  using GlobalPositionFunctor =
-      std::function<std::pair<Acts::Vector3, Acts::Vector2>(
-          const external_spacepoint_t&, float, float, float)>;
-
-  template <typename spacepoint_iterator_t>
+  template <typename spacepoint_iterator_t, typename callable_t>
   BinnedSPGroup<external_spacepoint_t>(
       spacepoint_iterator_t spBegin, spacepoint_iterator_t spEnd,
-      GlobalPositionFunctor toGlobal,
+      callable_t&& toGlobal,
```

Comment:
Is there a reason why the functor has to be generalised ?

Reply:
It's more a small optimization here that adds some additional flexibility. Being a universal reference we can simply pass an lvalue or rvalue and we do not need to create a `std::function`. 

Said so, the functionality is the same